Header header
# string hand_side
float32[] score
geometry_msgs/Point[] keypt
geometry_msgs/Point[] joint
float32[] angle
uint8[] gesture
geometry_msgs/Vector3 rvec
geometry_msgs/Vector3 tvec
float32 fps

